tests/ApiFilterTest.php

```diff
@@ -175,19 +177,50 @@ public function shouldParseQueryParametersAndApplyThemOneByOneToQueryBuilder(
         $this->assertSame($expectedPreparedValues, $preparedValues);
     }
 
+    /**
+     * @test
+     * @dataProvider provideInvalidQueryParameters
+     */
+    public function shouldNotParseQueryParameters(array $invalidQueryParameters, string $expectedMessage): void
```

Comment:
Misleading name - "shouldNotParseQueryParameters" is surely not intended behavior of the ApiFilter class. 
Maybe "Should throw exception on invalid query parameters"?

Reply:
I missed a `Invalid` in the name :+1: - it should have been `shouldNotParseInvalidQueryParameters` - exception is just a today solution of this, but it might change, so I dont want to tight name with throwing an exception - if it really tests, that invalid query parameters should not be parsed ..